lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "packages": ["apps/*", "packages/**", "scripts"],
+  "packages": ["apps/*", "packages/**", "scripts", "transforms"],
```

Comment:
Why is transforms at the same level as apps/packages/scripts? Could it go under packages?

Reply:
I put it top level since codemods aren't really a component or client code, so it felt odd to put it under packages. But I can move it.

Reply:
Moved under packages/codemod

Reply:
Thanks, I think I do prefer it there. Also means less changes to files like this one.